src/elements/flip-card/flip-card/flip-card.ts

```diff
@@ -0,0 +1,74 @@
+import type { CSSResultGroup, TemplateResult } from 'lit';
+import { html, LitElement } from 'lit';
+import { customElement, state } from 'lit/decorators.js';
+
+import { SbbLanguageController } from '../../core/controllers.js';
+import { i18nFlipCard, i18nReverseCard } from '../../core/i18n.js';
+import '../../button/secondary-button.js';
```

Comment:
Nitpick; I prefer to have the side-effect imports below the style import. What do you think?